github/ci/prow-deploy/kustom/base/configs/current/plugins/plugins.yaml

```diff
@@ -535,6 +542,12 @@ approve:
   lgtm_acts_as_approve: false
   ignore_review_state: true
   commandHelpLink: https://go.k8s.io/bot-commands
+- repos:
+  - kubevirt/secrets
+  require_self_approval: false
+  lgtm_acts_as_approve: true
```

Comment:
Hi @dhiller , why is there different handling of lgtm for the secrets repo versus the rest of the kubevirt repos?

Reply:
I was thinking that there's only a very small set of persons that is able to look at these PRs, so I thought this would make sense to differ here.